src/index.js

```diff
@@ -0,0 +1,47 @@
+import './style.css';
+import Todo from './modules/Todo.js';
+import saveTodo from './modules/saveTodo.js';
+import populateTodo from './modules/populateTodo.js';
+
+const mainContainer = document.querySelector('#todo-container');
+const clearCompleted = document.querySelector('.clear');
+
+const populateAllTodo = () => {
+  mainContainer.innerHTML = '';
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+
```

Comment:
Kindly use the DRY principle on this line of code,👍 you have repeated it at many places, and you can create a function and call it whenever you need it. your code will be straightforward to maintain and scalable🤩